### ISSUE TITLE
Bootstrap theme styling issues

### ISSUE THREAD

**Issue by JKingsnorth** [veda-consulting/uk.co.vedaconsulting.mosaico#526]
For the next release support for the 'crmstar' single page layout has been removed: https://github.com/veda-consulting-company/uk.co.vedaconsulting.mosaico/commit/ae6ffce10fd300ee0d8b4ddd2a671705787a27aa

However, there are a number of issues with the styling of the new default Bootstrap theme:

1 - Missing icon for 'Advanced' settings ✅

2 - Invisible titles and poor layout in Advanced settings screens

![image](https://user-images.githubusercontent.com/5212601/170715525-0c404187-1676-4100-8d1b-69e6b15e088b.png)

3 - Inconsistent styling and spacing of main input boxes for mailing name, recipients etc:

![image](https://user-images.githubusercontent.com/5212601/170715674-56628617-601a-4097-84b5-f5379a8ad607.png)

4 - Almost invisible text in hover state of primary action buttons ✅

![image](https://user-images.githubusercontent.com/5212601/170716093-3fff4733-e1fd-4bab-8fad-13f5d0807f71.png)

**Comment by mattwire** [comment]
@JKingsnorth Is this still an issue?

**Comment by JKingsnorth** [comment]
Yes @mattwire - still some styling issues in the 'Advanced settings' modal (see below). And the fields at the top of the page are still quite inconsistent. I've 'ticked' the two things in the original description which are fixed though.

![image](https://github.com/veda-consulting-company/uk.co.vedaconsulting.mosaico/assets/5212601/7932a392-056b-4664-b044-c50bb72fb8f8)
